src/d3a/models/area/__init__.py

```diff
@@ -144,6 +145,7 @@ def __init__(self, name: str = None, children: List["Area"] = None,
         self.redis_ext_conn = RedisMarketExternalConnection(self) \
             if external_connection_available and self.strategy is None else None
         self.should_update_child_strategies = False
+        self.matcher = MycoMatcher()
```

Comment:
Nit: I think the name `matcher` is pretty broad, so it's not immediately clear what it represents. Maybe `offers_bids_matcher` would be clearer (but quite long). Do you have any ideas about possible alternatives?

Reply:
I don't have any recommendation, I will then take `offers_bids_matcher ` as you proposed.